src/property-filter/__tests__/property-filter-controller.test.ts

```diff
@@ -12,6 +13,8 @@ import {
 } from '../interfaces';
 import { i18nStrings, toInternalProperties } from './common';
 
+const i18nStringsInternal: I18nStringsInternal = i18nStrings as any as I18nStringsInternal;
```

Comment:
That is a small hack to avoid transforming i18nStrings to i18nStringsInternal. The main purpose of the latter is to provide a more convenient interface for translation functions so that those can take internal token instead of formatted token. None of these functions is used in the below tests.